### PR TITLE
Add new analytics

### PR DESCRIPTION
**Pull request by cmcewen**
### Description

Add new segment integration

### CODE REVIEW

**Review by annakaz — APPROVED** [review]
Thanks for fixing this

File: packages/mobile/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
   <uses-sdk android:minSdkVersion="16" />
 
-  <application android:name=".MainApplication" android:allowBackup="true" android:fullBackupContent="@xml/backup_rules" android:icon="@mipmap/ic_launcher" android:roundIcon="@mipmap/ic_launcher_round" android:label="@string/app_name" android:theme="@style/AppTheme" android:networkSecurityConfig="@xml/network_security_config">
+  <application android:name=".MainApplication" android:allowBackup="false" android:icon="@mipmap/ic_launcher" android:roundIcon="@mipmap/ic_launcher_round" android:label="@string/app_name" android:theme="@style/AppTheme" android:networkSecurityConfig="@xml/network_security_config">
```
Comment (annakaz) [inline packages/mobile/android/app/src/main/AndroidManifest.xml:15]: How is backup related to this PR?

Comment (cmcewen) [reply packages/mobile/android/app/src/main/AndroidManifest.xml:15]: I ran into an issue with the manifest merging, and it's actually something we should have already disabled for security reasons

**Review by jeanregisser — APPROVED** [review]
Looks good!
Why AppsFlyer specifically?

**Pull request merged**